pkg/services/apiserver/builder/helper.go

```diff
@@ -245,6 +245,18 @@ func SetupConfig(
 
 	serverConfig.EffectiveVersion = v
 
+	// set priority for aggregated discovery
+	for _, b := range builders {
+		gvs := GetGroupVersions(b)
```

Comment:
Question / suggestion for the future – should we make sure that builders always return a single API group or do we have a use-case for multi-group builders?

If we enforce single version per builder, we could refactor `GetGroupVersions` to always return `group, versions` for simplicity & validation.